spec/features/after_transfering_an_ect/transfer_participant_steps.rb

```diff
@@ -0,0 +1,140 @@
+# frozen_string_literal: true
+
+module TransferParticipantSteps
+  include Capybara::DSL
+
+  def given_lead_provider_contracted_to_deliver_ecf
+    i = @lead_providers.length
+    lead_provider = create :lead_provider, :with_delivery_partner
+    cpd_lead_provider = create :cpd_lead_provider,
+                               lead_provider: lead_provider
+    create :ecf_statement, cpd_lead_provider: cpd_lead_provider
+
+    @lead_providers[i] = lead_provider
+    @lead_provider_tokens[i] = LeadProviderApiToken.create_with_random_token!(cpd_lead_provider: cpd_lead_provider)
+  end
+
+  def and_another_lead_provider_contracted_to_deliver_ecf
+    given_lead_provider_contracted_to_deliver_ecf
+  end
+
+  def and_sit_reported_programme(programme)
+    i = @sits.length
+    school = create :school,
+                    name: "#{programme} School #{i}",
+                    slug: "111111-#{programme}-school-#{i}",
+                    urn: "11111#{i}"
+    user = create(:user, full_name: "Carl Coordinator #{i}")
+    sit = create :induction_coordinator_profile,
+                 schools: [school],
+                 user: user
+    create :school_cohort, programme,
+           school: school,
+           cohort: @cohort
+
+    @privacy_policy.accept! sit.user
+    @sits[i] = sit
+  end
+
+  def and_another_sit_reported_programme(programme)
+    and_sit_reported_programme programme
+  end
+
+  def and_sit_reported_ect_participant(sit)
+    i = @participants.length
+    school_cohort = sit.schools.first.school_cohorts.first
+
+    participant = create :ect_participant_profile, school_cohort: school_cohort
+    @privacy_policy.accept! participant.user
+    @participants[i] = participant
+  end
+
+  def and_lead_provider_reported_partnership(lead_provider, school)
+    create :partnership,
+           school: school,
+           lead_provider: lead_provider,
+           delivery_partner: lead_provider.delivery_partners.first,
+           cohort: @cohort,
+           challenge_deadline: 2.weeks.ago
+  end
+
+  def and_another_lead_provider_reported_partnership(lead_provider, school)
+    and_lead_provider_reported_partnership lead_provider, school
+  end
+
+  def when_sit_takes_on_the_participant(sit, participant)
+    school = sit.schools.first
+    school_cohort = school.school_cohorts.first
+
+    participant.update!(school_cohort: school_cohort)
+    participant.teacher_profile.update!(school: school)
+  end
+
+  def then_participant_can_be_seen_by_lead_provider(lead_provider, lead_provider_token, participant)
+    assert_equal true, lead_provider.ecf_participant_profiles.include?(participant)
```

Comment:
Would be idea to avoid a mix of minitest/rspec assertions 
Could include  `include RSpec::Matchers` and then swap `assert_*` to the their RSpec equivalent: 
```suggestion
    expect(lead_provider.ecf_participant_profiles).to include(participant
```

Reply:
Changed in new commit